### ISSUE TITLE
SCM values are required, even if not needed

### ISSUE THREAD

**Issue by temp** [rocketeers/rocketeer#709]
I'm using rocketeer with rsync, but I'm still required to fill in the repository/username/password values in scm.php. Is there a way to disable this?

**Comment by Anahkiasen** [comment]
In theory if you set the values to `false` it won't ask you for them ever again IIRC

**Comment by Anahkiasen** [comment]
In theory if you set the values to `false` it won't ask you for them ever again IIRC

**Comment by temp** [comment]
Nope, neither false nor null have the desired effect. He always asks for the values.
Right now I'm using random values, but this feels a bit hacky ;-)

**Comment by Anahkiasen** [comment]
Can you try running `rocketeer flush` after changing the values to falsey ones in the config? I know Rocketeer 2 had some issues flushing the config cache sometimes

**Comment by temporarychaos** [comment]
Try setting deploy strategy to "sync" instead of "Sync."
